appveyor.yml

```diff
@@ -44,16 +46,19 @@ install:
   - npm install
 
 test_script:
+  - cmd: set PATH=%APPDATA%\npm;%PATH%
+  - cmd: set PATH=%APPDATA%\Roaming\npm;%PATH%
+  - cmd: npm install -g electron-prebuilt@%ELECTRON_VERSION%
```

Comment:
Yeah I think you're right. There's definitely some cleanup to do once this is working :+1: